components/expenses/PayoutBankInformationForm.js

```diff
@@ -457,12 +461,17 @@ const PayoutBankInformationForm = ({ isNew, getFieldName, host, fixedCurrency, i
   const currencyFieldName = getFieldName('data.currency');
   const selectedCurrency = get(formik.values, currencyFieldName);
 
-  const validateCurrencyMinimumAmount = () => {
+  const validateCurrency = () => {
     // Skip if currency is fixed (2) (3)
     // or if `availableCurrencies` is not set (but we're not supposed to be there anyway)
     if (fixedCurrency || !availableCurrencies) {
       return;
     }
+
+    if (!formik?.values?.currency) {
+      return 'Please select a currency';
```

Comment:
Intl. Check `formatFormErrorMessage(createError(ERROR.FORM_FIELD_REQUIRED))`

Reply:
Thanks for pointing it out. I've internationalized all the strings in this section, but I think we have to use `formatMessage` probably as these are validation messages not errors in my understanding. Let me know if you think otherwise. 🤔

Reply:
If you look at examples of `validate` functions for formik in their [docs](https://formik.org/docs/guides/validation), you'll see that they return an `errors` object. Thus we're calling them "form errors" here. I would keep calling them validation errors (not just "message") as they'll prevent you from submitting the form.

I believe the `formatFormErrorMessage` helper suggested above is pretty explicit about what it does: formatting form error messages. Maybe we can rename it to `formatFormValidationError` if that's not clear enough?
Anyway, if you look at the code, `formatFormErrorMessage` is a syntactic sugar that fallsback on `ERROR.FORM_FIELD_INVALID_VALUE` when the error is unknown. 

We have some form validation errors defined in https://github.com/opencollective/opencollective-frontend/blob/f4b15911a280d09cc38dddcea729dc53b207376c/lib/errors/index.js#L40, it's great if you can use them as is opens the road for unified error types between API and Frontend in the future.

Reply:
That being said, having a custom message here is nice too. No blocker with the `formatMessage({ defaultMessage: 'Please select a currency' });` approach you took here.